### PR TITLE
Feature/letter ownership

### PR DESCRIPTION
**Pull request by martinstannard**
@gogogarrett @Bockit

### CODE REVIEW

File: apps/skateboard/web/elm/GameBoard.elm [hunk 1]
```diff
@@ -176,18 +165,21 @@ playerView player =
     [ h3 [] [ text (player.name ++ " " ++ toString (player.score)) ] ]
 
 
-boardRow : Signal.Address Action -> List Letter -> Html
+boardRow : Signal.Address Action -> BoardRow -> Html
```
Comment (Bockit) [inline apps/skateboard/web/elm/GameBoard.elm]: :+1:

File: apps/skateboard/web/elm/GameBoard.elm [hunk 2]
```diff
@@ -154,7 +143,7 @@ view address model =
         , button
             [ disabled (hideSubmit model.candidate), Events.onClick address Submit ]
             [ text "Submit" ]
-        , h2 [] [ text (List.foldl (\c a -> a ++ c.letter) "" model.candidate) ]
+        , h2 [ class "candidate" ] [ text (List.foldl (\c a -> a ++ c.letter) "" model.candidate) ]
```
Comment (Bockit) [inline apps/skateboard/web/elm/GameBoard.elm]: Do you think it's a good idea to make this a helper function like you have letterStyle?

File: apps/skateboard/web/elm/Letter.elm [hunk 3]
```diff
@@ -0,0 +1,116 @@
+module Letter (..) where
+
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import List exposing (..)
+import Array exposing (..)
```
Comment (Bockit) [inline apps/skateboard/web/elm/Letter.elm]: Do you think it's worth making this explicit like GameBoard.elm? (and all maybe except the html and html.attributes)

**Comment by gogogarrett** [comment]
:ship:

File: apps/skateboard/web/elm/Letter.elm [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+type alias BoardRow =
+  List Letter
+
+
+type alias Board =
+  List BoardRow
+
+
+letterSurrounded : Letter -> Bool
```
Comment (Bockit) [inline apps/skateboard/web/elm/Letter.elm]: Comment here showing it's not really working

**Comment by Bockit** [comment]
:ship: when happy

**Pull request merged**